api/v1alpha1/clienttrafficpolicy_types.go

```diff
@@ -88,6 +88,11 @@ type ClientTrafficPolicySpec struct {
 	//
 	// +optional
 	Timeout *ClientTimeout `json:"timeout,omitempty"`
+	// PerConnectionBufferLimitBytes sets a soft limit on size of the cluster’s connections read and write buffers.
```

Comment:
it's not the cluster's connections, it's the listener... can you dig up the correct wording from envoy docs?

Reply:
This is actually the exact wording, maybe it's only available for upstreams?

Reply:
changed